bftengine/src/bftengine/ControllerBase.hpp

```diff
@@ -45,6 +45,7 @@ class ControllerBase {
   virtual void onMessage(const PreparePartialMsg* m) {}
 
   // TODO(GG): add more methods that may be useful  by controllers
```

Comment:
Can we remove this TODO? IMHO, it doesn't describe a concrete action item.

Reply:
it's Guy's todo I'm afraid to remove it :)